tests/test_hypothesis_plots.py

```diff
@@ -172,3 +182,86 @@ def test_plot_ridge(datatree, combined, plot_kwargs, labels_shade_label):
                 assert all(key in child for child in pc.viz.children.values())
         elif key not in ("remove_axis", "ticklabels"):
             assert all(key in child for child in pc.viz.children.values())
+
+
+ppc_kind_value = st.sampled_from(("kde", "cumulative"))
+ppc_group = st.sampled_from(("prior", "posterior"))
+ppc_observed = st.booleans()
+ppc_aggregate = st.booleans()
+ppc_sample_dims = st.sampled_from((["chain"], ["chain", "draw"]))
+ppc_facet_dims = st.sampled_from((["group"], ["hierarchy"], None))
+
+
+@st.composite  # composite func to determine num_pp_samples based on draws of group, sample_dims
+def draw_num_pp_samples(draw, group, sample_dims):
+    group = draw(group)
+    sample_dims = draw(sample_dims)
+    # print(f"\n sample_dims = {sample_dims}\ngroup = {group}")
+    chain_dim_length = 1 if group == "prior" else 3
+    draw_dim_length = 50 if sample_dims == ["chain", "draw"] else 1
+    total_num_samples = np.prod([chain_dim_length, draw_dim_length])
+
+    num_pp_samples = draw(st.integers(min_value=1, max_value=total_num_samples))
+    # print(f"\nnum_pp_samples = {num_pp_samples}")
+    return num_pp_samples
+
+
+@given(
+    plot_kwargs=st.fixed_dictionaries(
+        {},
+        optional={
+            "kind": plot_kwargs_value,
+            "predictive": plot_kwargs_value,
+            "observed": plot_kwargs_value,
+            "aggregate": plot_kwargs_value,
+            "observed_rug": plot_kwargs_value,
```

Comment:
I think none of these can be false, so they should not have their values be `plot_kwargs_value`, see what other tests like plot_forest are doing